synapse/rest/client/v1/push_rule.py

```diff
@@ -267,7 +267,7 @@ def _check_actions(actions):
         raise InvalidRuleException("No actions found")
 
     for a in actions:
-        if a in ["notify", "dont_notify", "coalesce"]:
+        if a in ["notify", "dont_notify", "coalesce", "org.matrix.msc2625.mark_unread"]:
```

Comment:
it'd be nice to extract some constants for the push rule actions.

Reply:
Agreed, but that'd make the diff a bit more hairy, so I'll do it as a separate PR.